Source/Cogworks.Umbraco.Essentials/ViewFiles/_ResponsiveImage.cshtml

```diff
@@ -0,0 +1,23 @@
+﻿@*
+    COpy this view to ~/Views/Partials/Shared/_ResponsiveImage.cshtml
```

Comment:
Remove this comment (https://github.com/thecogworks/Cogworks.Umbraco.Essentials/pull/18/files#r638489092)

Reply:
I added this comment in purpose, is important to tell how to work with this builder.